doc/start/documenting-ceph.rst

```diff
@@ -531,6 +531,264 @@ As noted earlier, you can make documentation contributions using the `Fork and
 Pull`_ approach.
 
 
+Squash Extraneous Commits
+-------------------------
+Each pull request ought to be associated with only a single commit. If you have
```

Comment:
suggest s/ought/should/

I've seen differing opinions on PRs that contain both code and docs:  some assert that docs and code should remain in separate commits.

I tend to amend commits and force-push them instead of squashing later, some people take issue with that, others demand it.

I think GitHub has the option to enable a "squash on merge" button for PRs, if we could get that turned on for `ceph/ceph` that might make this a lot easier.